pkg/nsx/services/staticroute/staticroute.go

```diff
@@ -133,3 +133,17 @@ func (service *StaticRouteService) ListStaticRoute() []model.StaticRoutes {
 	}
 	return staticRouteSet
 }
+
+func (service *StaticRouteService) Cleanup() error {
+	staticRouteSet := service.ListStaticRoute()
+	log.Info("cleaning up staticroute", "count", len(staticRouteSet))
+	for _, staticRoute := range staticRouteSet {
+		path := strings.Split(*staticRoute.Path, "/")
```

Comment:
Do we need to print every route path when deleting it

Reply:
done